documentation/jetty-documentation/src/main/asciidoc/programming-guide/server/sessions/session-sessioncache.adoc

```diff
@@ -133,11 +133,11 @@ include::../../{doc_code}/org/eclipse/jetty/docs/programming/server/session/Sess
 [[pg-server-session-customcache]]
 ===== Implementing a Custom SessionCache
 
-As previously mentioned, we highly recommend that you extend the link:{javadoc-url}/org/eclipse/jetty/server/session/AbstractSessionCache.html[AbstractSessionCache].
+As previously mentioned, we highly recommend that you extend the link:{javadoc-url}/org/eclipse/jetty/session/AbstractSessionCache.html[AbstractSessionCache].
```

Comment:
```suggestion
As previously mentioned, you should extend the link:{javadoc-url}/org/eclipse/jetty/session/AbstractSessionCache.html[AbstractSessionCache].
```

Reply:
I've modified the wording a little differently.